spec/acceptance/firewallchain_spec.rb

```diff
@@ -1,11 +1,6 @@
 require 'spec_helper_acceptance'
 
 describe 'puppet resource firewallchain command' do
-  before :all do
```

Comment:
this is normally needed for every test, so we can re-run the tests. why was it removed ?